media-source-respec.html

```diff
@@ -829,19 +829,17 @@
           <p>Follow these steps when <a def-id="duration"></a> needs to change to a <var>new duration</var>.</p>
           <ol>
             <li>If the current value of <a def-id="duration"></a> is equal to <var>new duration</var>, then return.</li>
-            <li>Set <var>old duration</var> to the current value of <a def-id="duration"></a>.
+			<li>If <var>new duration</var> is less than the highest start time reported by the <a def-id="buffered"></a> attribute across all <a>SourceBuffer</a> 
+			objects in <a def-id="sourceBuffers"></a>, then throw an <a def-id="invalid-state-error"></a> exception and abort these steps.</li>
+			  <p class="note">Duration reductions must first use <a def-id="remove"></a> to reduce the buffered range before updating <a def-id="duration"></a>.</p>
```

Comment:
s/Duration reductions must use/Duration reductions that would truncate currently buffered media are disallowed. To accomplish such a truncation and duration reduction, use/